### ISSUE TITLE
Sanitize throws MissingMethodException

### ISSUE THREAD

**Issue by jfren484** [mganss/HtmlSanitizer#68]
Before updating to the latest version of HtmlSanitizer that uses AngleSharp, HtmlSanitizer was working in my application. After updating, however, calling Sanitize throws an exception.  Here is how I'm using it (in an extension method):
```
        public static string Sanitize(this string htmlString)
        {
            var sanitizer = new HtmlSanitizer();
            return sanitizer.Sanitize(htmlString);
        }
```

And here are the details:
System.MissingMethodException was unhandled by user code
  HResult=-2146233069
  Message=Method not found: 'System.String AngleSharp.IMarkupFormattable.ToHtml(AngleSharp.IMarkupFormatter)'.
  Source=HtmlSanitizer
  StackTrace:
       at Ganss.XSS.HtmlSanitizer.Sanitize(String html, String baseUrl, IMarkupFormatter outputFormatter)
       at Tcbcsl.Presentation.Helpers.ExtensionMethods.Sanitize(String htmlString) in C:\Users\Jay\Documents\GitHubVisualStudio\Tcbcsl\Presentation\Helpers\ExtensionMethods.cs:line 119
       at lambda_method(Closure , NewsEditModel )
       at AutoMapper.Internal.DelegateBasedResolver`2.Resolve(ResolutionResult source)
       at AutoMapper.NullReferenceExceptionSwallowingResolver.Resolve(ResolutionResult source)
       at AutoMapper.PropertyMap.<>c.<ResolveValue>b__44_0(ResolutionResult current, IValueResolver resolver)
       at System.Linq.Enumerable.Aggregate[TSource,TAccumulate](IEnumerable`1 source, TAccumulate seed, Func`3 func)
       at AutoMapper.PropertyMap.ResolveValue(ResolutionContext context)
       at AutoMapper.Mappers.TypeMapObjectMapperRegistry.PropertyMapMappingStrategy.MapPropertyValue(ResolutionContext context, Object mappedObject, PropertyMap propertyMap)
  InnerException:

**Comment by 304NotModified** [comment]
That's a breaking change in AngleSharp (again)

**Comment by mganss** [comment]
Published a new release (3.2.105).

@304NotModified I've left it as is for now, hoping that they won't change their API again for 1.x. Should this happen again, I'll require a specific version.
